packages/api-explorer/src/components/Response/ResponseBody.jsx

```diff
@@ -51,10 +51,11 @@ Unauthorized.defaultProps = {
 
 // eslint-disable-next-line react/prop-types
 function ResponseBody({ result, isOauth, oauth, isCollapsed}) {
+  const { status, responseBody } = result;
   return (
     <div style={{ display: 'block', color: '#fff'}}>
-      {result.status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />}
-      {result.responseBody ?
+      {status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />}
+      {responseBody !== undefined && responseBody !== null ?
```

Comment:
I think that you should check only undefined, null is a value that you want return.

Reply:
The previous implementation makes the null value correspond to "The response has no body". Do we want to change this?

Reply:
IMO, yes.
@davidebianchi what do you think about it?

Reply:
The client cast an empty body to a null responseBody value: